src/Kentico.Xperience.Lucene.Admin/LuceneConfigurationModel.cs

```diff
@@ -19,55 +29,92 @@ public class LuceneConfigurationModel
     [MinLength(1)]
     public string IndexName { get; set; } = "";
 
-    [LuceneIndexConfigurationComponent(Label = "Included Paths", Order = 2)]
+
+    [Obsolete("The property is not used anymore. Use IncludedPaths on items of the Channels collection instead.")]
     public IEnumerable<LuceneIndexIncludedPath> Paths { get; set; } = Enumerable.Empty<LuceneIndexIncludedPath>();
 
+
+    /// <summary>
+    /// The configurations for different website channels associated with the index.
+    /// </summary>
+    [LuceneIndexConfigurationComponent(Label = "Configured Channels", Order = 2)]
+    public IEnumerable<LuceneIndexChannelConfiguration> Channels { get; set; } = Enumerable.Empty<LuceneIndexChannelConfiguration>();
+
+
+    /// <summary>
+    /// The reusable content types that the index will include.
+    /// </summary>
     [GeneralSelectorComponent(dataProviderType: typeof(ReusableContentOptionsProvider), Label = "Included Reusable Content Types", Order = 3)]
     public IEnumerable<string> ReusableContentTypeNames { get; set; } = Enumerable.Empty<string>();
 
+
+    /// <summary>
+    /// The languages that the index will support.
+    /// </summary>
     [GeneralSelectorComponent(dataProviderType: typeof(LanguageOptionsProvider), Label = "Indexed Languages", Order = 4)]
     public IEnumerable<string> LanguageNames { get; set; } = Enumerable.Empty<string>();
 
-    [DropDownComponent(Label = "Channel Name", DataProviderType = typeof(ChannelOptionsProvider), Order = 5)]
-    public string ChannelName { get; set; } = "";
```

Comment:
Removing public property is considered as a breaking change.
Bring the property back and mark it with Obsolete attribute.